webrtc.html

```diff
@@ -7472,7 +7472,8 @@ <h2>
       </p>
       <p>
         Following the rules in <span data-jsep="imageattr">[[!RFC8829]]</span>,
-        the video MAY be downscaled in order to fit the SDP constraints. The
+        the video MAY be downscaled in order to fit the SDP constraints and
+        alignment limitations of the encoder. The
```

Comment:
What about downscaling due to cpu overuse detection or BWE?

Reply:
I'll move mention of these to the paragraph above, and restore this paragraph to not disturb it, as it appears to have a lot of history.